src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -625,6 +625,13 @@ public virtual IEnumerable<CallStackFrame> GetCallStack()
         public virtual Breakpoint GetBreakpoint(int id) =>
```

Comment:
@rjmholt, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/building/internals.md.https://cmake.org/download/ should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 500 (Internal Server Error).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```